store/postgres/replay_repository.go

```diff
@@ -83,13 +141,23 @@ func (repo *replayRepository) Insert(replay *models.ReplaySpec) error {
 
 func (repo *replayRepository) GetByID(id uuid.UUID) (models.ReplaySpec, error) {
 	var r Replay
-	if err := repo.DB.Where("id = ?", id).Find(&r).Error; err != nil {
+	if err := repo.DB.Where("id = ?", id).Preload("Job").Preload("Job.Project").
+		Preload("Job.Project.Secrets").Find(&r).Error; err != nil {
```

Comment:
Can we skip loading Projects & secrets? Sorry, I can't find where we are using them?

Reply:
we are using them in scheduler, since we need scheduler auth token and scheduler host to get dag run status. should i just load them just before calling the scheduler then? in replay manager?

Reply:
updated to not preload projects and secrets, since in replay syncer we fetched all the projects and when getting replay, we are getting it from the user